docs/source/virtual_rainforest/core/config.md

```diff
@@ -50,6 +50,21 @@ optional. A list of modules to be configured can be specified using the tag
 critically fail. If this tag isn't provided the default set of modules will be loaded,
 i.e. the standard versions of `animals`, `plants`, `soil`, and `abiotic`.
 
+## Model constants
+
+As well as general options to the setup the models, the configuration will contain
+constants. These will need to be varied in order for sensitivity analysis to be carried
+out. This means that they have to be specified in a standard way so that the sensitivity
+analysis can find and alter them straightforwardly. Each model should define its own set
+of constants under `model_name.constants`. Within this one or more groups of constants
+should be defined, these groups are each input for a particular dataclass in the relevant
+`constants.py` module, and should use the same name as this dataclass. Individual
+constants should then be defined within these groups. The full tag structure would take
+the following pattern `model_name.constants.SomeVars.constant_1`. It is important to
+note that **only** the constants you wish to change need to be included in the
+configuration, any constant that isn't given will just use the default value set in the
+relevant `constants.py` module.
+
```

Comment:
Should we mention here the `core.constants` or will this be considered separately?

Reply:
I don't think that any of this will be different for `core.constants`, provided we are prepared to call `core` a model name. Unless we have to settle on a radically different way of handling `core` constants

Reply:
I think it will as `core` isn't a model. This doesn't matter until we actually move shared constants into core but I think the core configuration will use a different mechanism. It isn't immediately clear to me where those config instances end up, but it's part and parcel of restructuring the `Data`/`Grid`/`Config` information.